knowledge-base/inputs-open-programmatically.md

```diff
@@ -42,16 +42,15 @@ This Knowledge Base article covers multiple scenarios:
 * How to expand the item list when the ComboBox input is focused via tab?
 * How to open the MultiSelect popup from code?
 * How to open the MultiSelect list during keyboard navigation tabbing?
-* Is it possible to open the MultiSelect item popup open on page load, without clicking?
 * How to show the Calendar or Time popup immediately when the user focuses the Date/Time Picker textbox?
 
 ## Solution
 
-To begin with, [all Telerik inputs and dropdowns have a `FocusAsync` method]({%slug inputs-kb-focus%}) to focus them programmatically.
+To begin with, all Telerik inputs and dropdowns have an `Open` method.
 
-The **DropDownList** and **MultiSelect** open automatically on click. They need JavaScript code only to open on tab and `FocusAsync`.
+The **DropDownList** and **MultiSelect** open automatically on click. They need JavaScript code only to open on focus.
```

Comment:
Replace the bold font in the component names with a regular font on all occurrences. Bold in this context doesn't comply with the Style Guide. See [Brand Names](https://docs.telerik.com/style-guide/brandnames) and [Formatting](https://docs.telerik.com/style-guide/element-formatting).